pages/nft-book-store/send/[classId].vue

```diff
@@ -259,8 +260,12 @@ function handleClickEditNFTId () {
 async function fetchNFTMetadata () {
   try {
     isVerifyingNFTId.value = true
+    if (!nftIds.value.length || !nftIds.value[0]) {
+      nftImage.value = ''
+      return
+    }
     try {
-      const data = await $fetch(`${LCD_URL}/cosmos/nft/v1beta1/nfts/${classId.value}/${nftId.value}`)
+      const data = await $fetch(`${LCD_URL}/cosmos/nft/v1beta1/nfts/${classId.value}/${nftIds.value[0]}`)
```

Comment:
Should we need to check all the ids?

Reply:
this is mainly for fetching image preview